homeassistant/components/sensor/google_travel_time.py

```diff
@@ -24,14 +24,18 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_TRAFFIC_MODEL = 'traffic_model'
+ATTR_AVOID = 'avoid'
+
 CONF_DESTINATION = 'destination'
 CONF_OPTIONS = 'options'
 CONF_ORIGIN = 'origin'
 CONF_TRAVEL_MODE = 'travel_mode'
+CONF_UPDATE_INTERVAL = 'update_interval'
```

Comment:
This is not needed. Neither is the Throttle. Home Assistant has already built in support for users to configure their update interval: https://home-assistant.io/docs/configuration/platform_options/#scan-interval

The default scan interval is set by definng the SCAN_INTERVAL constant.